google/cloud/storage/object_metadata.cc

```diff
@@ -83,7 +83,7 @@ ObjectMetadata ObjectMetadata::ParseFromString(std::string const& payload) {
   return ParseFromJson(json);
 }
 
-std::string ObjectMetadata::ToJsonString() const {
+std::string ObjectMetadata::UpdatePayload() const {
```

Comment:
At first glance, I read this function name semantically as "Update the payload", which made no sense.  Maybe `JsonUpdatePayload` or `JsonPayloadForUpdate`?

Reply:
Fixed.